src/modules/advertising.json

```diff
@@ -94,6 +109,14 @@
     },
     {
       "name": "advertisingId",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-uses": [
+            "xrn:firebolt:capability:advertising:configuration"
```

Comment:
We are actually going through this with badger, but the advertising team would like to give some apps access to advertising configuration but not access to the advertisingId. So Maybe this should be a different capability

Reply:
I agree, but the ifa is in the configuration, so there's really no way to make two capabilities w/out changing that.

Reply:
fixed